### ISSUE TITLE
why set model mode as train() instead of eval() in test?

### ISSUE THREAD

**Issue by ningbende** [Lotayou/Face-Renovation#31]
I noticed the comment ,but why ?
   ### 20200218 Critical Bug
    # When model is set to eval mode, the generated image
    # is not enhanced whatsoever, with almost 0 residual
    # when turned to training mode, it behaves as expected.
    ###
    #model.eval()
    model.netG.eval()
    #model.netG.train()

**Comment by Lotayou** [comment]
See #12 

> ```
> but I notice that you have mentioned a bug in your code that the predicted images are not shown as expected when we set the model to eval model.
> ```
> 
> 
> This is a Pytorch [bug](https://discuss.pytorch.org/t/spectral-norm-in-eval-mode/26223) where the spectral normalization layer in SPADE is not working under eval mode in dataparallel. A possible fix is suggested at [here](https://github.com/pytorch/pytorch/pull/12671)
